test/common/tracing/http_tracer_impl_test.cc

```diff
@@ -336,6 +336,7 @@ TEST(HttpConnManFinalizerImpl, SpanPopulatedFailureResponse) {
   EXPECT_CALL(span, setTag(Eq("cc"), Eq("c")));
   EXPECT_CALL(config, requestHeadersForTags());
   EXPECT_CALL(config, verbose).WillOnce(Return(false));
+  EXPECT_CALL(config, maxPathTagLength).WillOnce(Return(256));
```

Comment:
Would it be worth setting the length to something lower and check that the `http.url` tag actually has a truncated value?

Reply:
Yeah that is fine.